definitions/variable/sdg-indicators/sdg.yaml

```
@@ -31,6 +31,12 @@
       gas)
     unit: million
     navigate: Population|Clean cooking Access
+- Population|Sanitation Access:
+    description: Population with access to improved sanitation
+    unit: million
```

Sorry, just realized that this variable actually did exist, just misspelled..
```suggestion
    unit: million
    navigate: Population|Sanitation Acces
    engage: Population|Sanitation Acces
```